### PR TITLE
Added method isSchemaDropDownDisabled() for handling error when updat…

### PR DESCRIPTION
**Pull request by anezboretskiy**
…e schema

- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Added method isSchemaDropDownDisabled() 

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [x] No need to
- [ ] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaEditView.java [hunk 1]
```diff
@@ -23,10 +24,10 @@ public SchemaEditView selectSchemaTypeFromDropdown(SchemaCreateView.SchemaType s
         return this;
     }
     @Step
-    public SchemaEditView selectCompatibilityLevelFromDropdown(CompatibilityLevel.CompatibilityEnum level) {
+    public static SchemaEditView selectCompatibilityLevelFromDropdown(CompatibilityLevel.CompatibilityEnum level) {
         $x("//ul[@name='compatibilityLevel']").click();
         $x("//li[text()='" + level.getValue() + "']").click();
-        return this;
+        return new SchemaEditView();
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaEditView.java:30]: no need to do this way. you can call method from this class using 'new SchemaEditView()'

**Review by VladSenyuta — CHANGES_REQUESTED** [review]

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaEditView.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
 public class SchemaEditView {
 
     SelenideElement newSchemaTextArea = $("#newSchema [wrap]");
+    private static final SelenideElement schemaTypeDropDown = $(By.xpath("//ul[@name='schemaType']"));
```
[the remark this answers is not in the document]
Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaEditView.java:18]: the best practice in this case to use 'protected' modifier only

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaEditView.java (hunk 1 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaEditView.java]: Done)

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaEditView.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaEditView.java]: corrected

**Review by VladSenyuta — APPROVED** [review]

**Pull request merged**